server/src/main/java/org/elasticsearch/reservedstate/service/FileSettingsService.java

```diff
@@ -127,6 +128,15 @@ protected void processFileChanges() throws ExecutionException, InterruptedExcept
         completion.get();
     }
 
+    @Override
+    protected void onProcessFileChangesException(Exception e) {
+        if (e instanceof ExecutionException && e.getCause() instanceof FailedToCommitClusterStateException f) {
+            logger.error("Unable to commit cluster state", e);
```

Comment:
Oh right, I forgot about the `f` here. Initially I was just logging `f` but decided that more context never hurts so I just logged the original exception `e`.

Reply:
You're still creating a variable `f` which is not referenced